adapters/latest/src/main/java/org/apache/cassandra/sidecar/adapters/latest/CassandraClusterMembershipOperations.java

```diff
@@ -16,17 +16,17 @@
  * limitations under the License.
  */
 
-package org.apache.cassandra.sidecar.cassandra40;
+package org.apache.cassandra.sidecar.adapters.trunk;
```

Comment:
See my previous comment about the "default" or "latest" implementation vs. forking a new implementation for some older version that doesn't support something. Calling this 40 doesn't make sense because it is, essentially, the _default_ implementation. Again, with 3.11 also in the codebase this would have been clearer.